spec/factories.rb

```diff
@@ -1,2 +1,6 @@
 FactoryGirl.define do
+  factory :book do
+    title "The Golden Compass"
```

Comment:
https://github.com/margaretoti/library-network/pull/1/files#r107217159 I suggested you add a unique constant to title. If you do that you'll need to use `sequence` which is a FactoryGirl method that makes it easy to generate unique sequences